vnext/Shared/HermesRuntimeHolder.cpp

```diff
@@ -29,6 +31,55 @@ namespace {
 std::unique_ptr<facebook::hermes::HermesRuntime> makeHermesRuntimeSystraced(
     const ::hermes::vm::RuntimeConfig &runtimeConfig) {
   SystraceSection s("HermesExecutorFactory::makeHermesRuntimeSystraced");
+
+  auto res = LoadLibrary(L"hermes.dll");
+  if (!res) {
+    DWORD lastError = GetLastError();
+    std::stringstream ostr;
+    ostr << "Failure loading hermes.dll:" << std::hex << lastError;
+    facebook::react::tracing::error(ostr.str().c_str());
+  } else {
+    facebook::react::tracing::error("hermes.dll loaded successfully");
+  }
+
+  auto res2 = LoadLibraryEx(L"hermes.dll", NULL, DONT_RESOLVE_DLL_REFERENCES);
```

Comment:
>LoadLibraryEx [](http://example.com/codeflow?start=14&length=13)

What if it is already loaded above? Why do we try it again?

Reply:
this seems dangerous, since this flag a) doesn't load dependencies of hermes (so it's not safe to call into hermes just yet), and b) bypasses calling dllmain which might be needed to set up / initialize

Reply:
As stated earlier.. all these are diagnostics hence please ignore ..